manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+    "author": "WebThingsIO",
+    "description": "Support for W3C compliant native web things",
+    "gateway_specific_settings": {
+        "webthings": {
+            "exec": "{nodeLoader} {path}",
+            "primary_type": "adapter",
+            "strict_max_version": "*",
+            "strict_min_version": "0.10.0",
```

Comment:
The gateway provides the `gateway-addon` dependency to the addons.
If we use version `1.1.0-alpha.2` of the `gateway-addon` this needs to be something like `1.1.0`.
Otherwise the types won't match.

Reply:
The other option would be to use `1.0.0` in both cases if it should run on the current 1.0 release of the gateway.